### ISSUE TITLE
Don't draw a fake coastline in the border of unexplored water.

### ISSUE THREAD

**Issue by calloatti** [Hunternif/AntiqueAtlas#236]
antiqueatlas-5.1.0-fabric

Right now AA does this on the border of unexplored water bodies:

![image](https://user-images.githubusercontent.com/12817862/61256584-b0a8a300-a743-11e9-9c9e-19a6f53ce913.png)

So basically there is no way to know if some coastlines are real coastlines of just the border of unexplored areas.

Maybe it would be better to do something like this:

![image](https://user-images.githubusercontent.com/12817862/61256628-d33abc00-a743-11e9-8140-5759fb7e1cc5.png)

So basically treat the border between water and unexplored areas the same way as the border between any other terrain and unexplored areas without drawing fake coastlines.

**Comment by Hunternif** [comment]
This make a ton of sense. It's just a technical limitation of the way tiles work right now.
